stock-service/src/main/java/rs/banka4/stock_service/config/RetrofitConfig.java

```diff
@@ -0,0 +1,31 @@
+package rs.banka4.stock_service.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import okhttp3.OkHttpClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import retrofit2.Retrofit;
+import retrofit2.converter.jackson.JacksonConverterFactory;
+
+@Configuration
+public class RetrofitConfig {
+
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
spring has a well configured and consistent with the rest of the API object mapper - do not make a new one